### ISSUE TITLE
NativeScript build error: 'FormlyNativescriptModule' does not appear to be an NgModule class

### ISSUE THREAD

**Issue by timbell** [ngx-formly/ngx-formly#3385]
I've just updated my NativeScript project and I'm now getting this build error:
```
ERROR in src/app/app.module.ts:14:96 - error NG6002: 'FormlyNativescriptModule' does not appear to be an NgModule class.

14   imports: [NativeScriptModule, AppRoutingModule, ReactiveFormsModule, FormlyModule.forRoot(), FormlyNativescriptModule],
                                                                                                  ~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@ngx-formly/nativescript/lib/ui-nativescript.module.d.ts:1:22
    1 export declare class FormlyNativescriptModule {
                           ~~~~~~~~~~~~~~~~~~~~~~~~
    This likely means that the library (@ngx-formly/nativescript) which declares FormlyNativescriptModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.
```
This error was not present previously. The versions where it worked were:
- Angular 13.1.1
- NativeScript 8.1.5
- Formly 6.0.0-beta.2

And the versions now being used are:
- Angular 14.0.6
- NativeScript 8.3.0
- Formly 6.0.0-rc.1

To reproduce, do the following:
- create a NativeScript test project: `ns create test --template @nativescript/template-hello-world-ng`
- add Formly to the project: `npm install @ngx-formly/core@6.0.0-rc.1 @ngx-formly/nativescript@6.0.0-rc.1 --save`
- import the Formly modules in the project's `app.module.ts`:
```
import { NgModule, NO_ERRORS_SCHEMA } from '@angular/core'
import { ReactiveFormsModule } from '@angular/forms'
import { NativeScriptModule } from '@nativescript/angular'
import { FormlyModule } from "@ngx-formly/core"
import { FormlyNativescriptModule } from '@ngx-formly/nativescript'

import { AppRoutingModule } from './app-routing.module'
import { AppComponent } from './app.component'
import { ItemsComponent } from './item/items.component'
import { ItemDetailComponent } from './item/item-detail.component'

@NgModule({
  bootstrap: [AppComponent],
  imports: [NativeScriptModule, AppRoutingModule, ReactiveFormsModule, FormlyModule.forRoot(), FormlyNativescriptModule],
  declarations: [AppComponent, ItemsComponent, ItemDetailComponent],
  providers: [],
  schemas: [NO_ERRORS_SCHEMA],
})
export class AppModule {}
```
- build the project: `ns build android`

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.0.0-rc.2](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-rc.2) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by timbell** [comment]
Yes, that fixes it for me. Thanks!
